Dockerfile

```diff
@@ -1,4 +1,4 @@
-FROM node:13.13.0
+FROM node:13.13.0-alpine3.10
```

Comment:
Never use unstable NodeJS version unless you are testing something

Reply:
You recommend to use `LTS` versions? or what do you mean by unstable?

Reply:
LTS versions are stable and latest versions are unstable

Reply:
TY!